### PR TITLE
Release 2021.05.1

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** Adding Release Notes for 2021.05.1
- **Is this PR related to a Monolith release?** Yes. Release Notes for 2021.05.1

### Resolves:

* [OKTA-393671](https://oktainc.atlassian.net/browse/OKTA-393671)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 1]
```diff
@@ -401,7 +401,7 @@ Based on the type of token and whether it is active, the returned JSON contains
 | Error Id          | Details                                                                                                  |
 | :---------------- | :------------------------------------------------------------------------------------------------------- |
 | invalid_client    | The specified `client_id` wasn't found.                                                                  |
-| invalid_request   | The request structure was invalid. For example, the basic authentication header was malformed, both header and form parameters were used for authentication, or no authentication information was provided. |
+| invalid_request   | The request structure was invalid. For example, the basic authentication header was malformed, both header and form parameters were used for authentication, no authentication information was provided, or the request contains duplicate parameters. |
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:404]: were used > are used
was provided > is provided

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 2]
```diff
@@ -401,7 +401,7 @@ Based on the type of token and whether it is active, the returned JSON contains
 | Error Id          | Details                                                                                                  |
 | :---------------- | :------------------------------------------------------------------------------------------------------- |
 | invalid_client    | The specified `client_id` wasn't found.                                                                  |
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:403]: General comment: Should the sentences in the Details column be in the present tense?
wasn't found > isn't found
was invalid > is invalid

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 3]
```diff
@@ -299,7 +299,7 @@ Based on the scopes requested. Generally speaking, the scopes specified in a req
 | :----------------------- | :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | invalid_client           | The specified `client_id` wasn't found.                                                                                                                                                                      |
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:300]: General comment: Should the sentences in the Details column be in the present tense?
wasn't found > isn't found
was invalid > is invalid

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 4]
```diff
@@ -484,7 +484,7 @@ A successful revocation is denoted by an HTTP 200 OK response. Note that revokin
 | Error Id          | Details                                                                                                                                                       |
 | :---------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | invalid_client    | The specified `client_id` wasn't found.                                                                                                                       |
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:486]: General comment: Should the sentences in the Details column be in the present tense?
wasn't found > isn't found
was invalid > is invalid

Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:486]: General comment: Should the sentences in the Details column be in the present tense?
wasn't found > isn't found
was invalid > is invalid

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md [hunk 5]
```diff
@@ -4,6 +4,30 @@ title: Okta API Products Release Notes 2021
...
+| [Okta Sign-In Widget, version 5.6.1](#okta-sign-in-widget-version-5-6-1) | May 12, 2021 |
+| [Bugs fixed in 2021.05.1](#bugs-fixed-in-2021-05-1)         | May 12, 2021           |
+
+#### Okta Sign-In Widget, version 5.6.1
+
+For details about this release, see the Okta [Sign-In Widget Release Notes](https://github.com/okta/okta-signin-widget/releases/tag/okta-signin-widget-5.6.1). For more information about the Widget, see the Okta [Sign-In Widget Guide](/code/javascript/okta_sign-in_widget/). <!--OKTA-393866-->
+
+#### Bugs fixed in 2021.05.1
+
+* Duplicate parameter names were passed in requests to the following [OAuth endpoints](/docs/reference/api/oidc/#possible-errors), and no error message was sent. (OKTA-132318)
```
Comment (christinechoy-okta) [inline packages/@okta/vuepress-site/docs/release-notes/2021/index.md:20]: Suggested change:
Duplicate parameter names were passed when requests were made to the following OAuth endpoints...

**Review by christinechoy-okta — COMMENTED** [review]
Just a few comments. Great job!

File: packages/@okta/vuepress-site/docs/release-notes/2021/index.md (hunk 5 above)
Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/release-notes/2021/index.md:20]: @christinechoy-okta - the duplicate parameter names were passed IN the requests, not when requests were made. So, the original sentence is more accurate.

**Review by christinechoy-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/reference/api/oidc/index.md [hunk 6]
```diff
@@ -297,9 +297,9 @@ Based on the scopes requested. Generally speaking, the scopes specified in a req
...
+| invalid_client           | The specified `client_id` isn't found.                                                                                                                                                                      |
 | invalid_grant            | The `code`, `refresh_token`, or `username` and `password` combination is invalid, or the `redirect_uri` doesn't match the one used in the authorization request.                                          |
-| invalid_request          | The request structure was invalid. For example: the basic authentication header is malformed, both header and form parameters were used for authentication, or no authentication information was provided. |
+| invalid_request          | The request structure is invalid. For example, the basic authentication header is malformed, both header and form parameters are used for authentication, no authentication information is provided, or the request contains duplicate parameters. |
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/api/oidc/index.md:302]:
```suggestion
| invalid_request          | The request structure was invalid. For example, the basic authentication header is malformed, both header and form parameters are used for authentication, no authentication information is provided, or the request contains duplicate parameters. |
```

**Pull request merged**